### DISCUSSION TITLE
How to ensure menu item order doesn't get reversed?

### DISCUSSION THREAD

**Discussion by owengretzinger** [nandorojo/zeego#117] [Q&A] [answered]
(Using iOS)

When I click a menu trigger on the top half of the screen, it works as expected:

<img width="215" alt="image" src="https://github.com/user-attachments/assets/7c3f268a-a58a-4b86-8201-36df0b394d2e">

But when I click one on the bottom half of the screen, the menu items are reversed:

<img width="208" alt="image" src="https://github.com/user-attachments/assets/0ba967ba-a69b-432f-9a2e-58d9479c563f">

I haven't found anything in the documentation or online about this. Is this the intended behaviour? Is there any way to ensure the items always show up in the same order?

**Comment by nandorojo** [comment, marked as answer]
This is how iOS works: the order determines how close to the trigger the items are

**Reply by owengretzinger** [reply]
Dang. And there's no way to change that?

**Reply by owengretzinger** [reply]
Managed to find some stuff about it on iOS, I'll look into it a bit more. Thanks for your answer!
